docs/installing-solidity.rst

```diff
@@ -99,7 +101,8 @@ We are also releasing a `snap package <https://snapcraft.io/>`_, which is instal
 
     sudo snap install solc
 
-Or if you want to help testing the unstable solc with the most recent changes from the development branch:
+If you want to help testing the unstable solc with the most recent changes
```

Comment:
``the unstable version of solc`` or ``the current development version of solc``?